compiler/damlc/tests/daml-test-files/AuthFailure.daml

```diff
@@ -92,6 +96,18 @@ t4_LookupByKeyMissingAuthorization = scenario do
     lookupByKey @TheContract4 ([alice],100)
     abort "t4 finished"
 
+Why does this fail? It doesn't fail when converted to scripts, as from my perspective - shouldn't.
+bob is explicitly an observer of the contract with that key, why should he not be able to look it up?
+-}
```

Comment:
Unsure whats happening with this test, would love some clarity

Reply:
Why should it fail?

Reply:
It is important in the test you do not change fetch / lookup by a query which work very differently.